modules/abstract-lightning/src/wallet/lightning.ts

```diff
@@ -94,6 +98,37 @@ export interface ILightningWallet {
    * @returns {Promise<unknown>} A promise resolving to the updated wallet response or throwing an error if the update fails.
    */
   updateWalletCoinSpecific(params: UpdateLightningWalletClientRequest): Promise<unknown>;
+
```

Comment:
Arrange the functions in following order, so callers will be able to reason it easily. Same goes for Express APIs.

getLightningKeychain
getLightningAuthKeychains
updateWalletCoinSpecific
createInvoice
getInvoice
listInvoices
payInvoice
getPayment
listPayments
getTransaction
listTransactions
getChannelBackup